_posts/patterns/2015-04-15-icons.md

```diff
@@ -411,7 +411,7 @@ info:               Icons are used to clearly and visually represent some inform
     <div class="example example-icon">
         <div class="icon-display">
             <span class="icon-fallback icon-fallback-text">
-                <span class="icon icon-exclamation-triangle" aria-hidden="true"></span>
+                <span class="icon icon-warning" aria-hidden="true"></span>
                 <span class="text">Warning</span>
             </span>
         </div>
```

Comment:
Same comment on updating to lowercase for [419](https://github.com/edx/ux-pattern-library/pull/284/files#diff-8e6a0a47e453c2074d1a3b7fbd3c0280R419).